spec/factories/access_control_permission.rb

```diff
@@ -0,0 +1,15 @@
+FactoryBot.define do
+  factory :access_control_permission, :traits => [:has_tenant]
+
+  trait :has_read_permission do
+    permission
+  end
+
+  trait :has_update_permission do
+    permission { Permission.create!(:name => 'update') }
```

Comment:
@syncrou Should this be find_or_create_by!

Reply:
@syncrou I think you can actually just do this, maybe?:
`permission { {:name => 'update'} }`

Reply:
@mkanoor possibly it should be.  @eclarizio I tried your suggestion and it didn't work because these are in the `access_control_permission` factory.
My question:
This does work:
```
permission { FactoryBot.create(:permission, :name => 'update') }
```
But along the lines of what @mkanoor is suggesting, it does create a new entry in the db each time it is called. Is it still better to use the `FactoryBot` way? Or simply call `find_or_create_by!` off the AR model?

Reply:
Does `association :permission, :name => 'update'` work? I don't know if that solves the issue of creating multiple permissions in the database, though.